lib/taskana-cdi/src/test/java/pro/taskana/TaskanaRestTest.java

```diff
@@ -23,12 +29,12 @@
 	private TaskanaEjb taskanaEjb;
 
 	@GET
-	public Response startTask() throws NotAuthorizedException, WorkbasketNotFoundException, ClassificationNotFoundException {
+	public Response startTask() throws NotAuthorizedException, WorkbasketNotFoundException, ClassificationNotFoundException, ClassificationAlreadyExistException {
 		Workbasket workbasket = new Workbasket();
 		workbasket.setName("wb");
 		taskanaEjb.getWorkbasketService().createWorkbasket(workbasket);
-		Classification classification = new Classification();
-		taskanaEjb.getClassificationService().addClassification(classification);
+		Classification classification = (Classification) new ClassificationImpl();
```

Comment:
We should behave like a client in this situation. I.e. don't use ClassificationImpl but create a classification via ClassificationService.newClassification().

Reply:
I will search and fix all classification-casts which are like this. The instantiation by service-call will be added in the upcoming pull-request.